config/config.go

```diff
@@ -843,6 +848,25 @@ func (c *Config) Valid() error {
 		}
 	}
 
+	// check capacity and the quota when OOMUseTmpStorage is enabled
+	if c.OOMUseTmpStorage {
+		if c.TempStorageQuota < 0 {
+			// print warning
+		} else {
+			capacityByte, err := linux.GetTargetDirectoryCapacity(c.TempStoragePath)
+			if err != nil {
+				if err.Error() == "Get directory capacity not supported in non-linux system yet" {
+					// print warning
+				} else {
+					return err
+				}
+			}
+			if int64(capacityByte) > c.TempStorageQuota {
+				return fmt.Errorf("value of [temp-storage-quota] exceeds the capacity of the [temp-storage-path] directory")
```

Comment:
It would be better to print the path of `c.TempStoragePath` in this error message.